packages/design-system/src/styles/components/_Badge.scss

```diff
@@ -2,6 +2,26 @@
 
 $badge-font-size-big: $base-font-size;
 
+@mixin badge-high-contrast-mode-remove-color() {
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    background-color: windowText;
+    color: window;
+    -ms-high-contrast-adjust: none;
+  }
+  /* stylelint-enable scss/media-feature-value-dollar-variable */
+}
+
+@mixin badge-high-contrast-mode-label($label) {
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    &::before {
+      content: $label + ': ';
+    }
+  }
+  /* stylelint-enable scss/media-feature-value-dollar-variable */
+}
+
```

Comment:
Is this still necessary if the label is now being added in markup?

Reply:
No, it's not needed. I've removed all the `@mixin` style relating to label.